packages/permissions/README.md

```diff
@@ -1,3 +1,7 @@
 # `@thorgate/spa-permissions`
 
 > Permissions helpers used by [Thorgate project template SPA variant](https://gitlab.com/thorgate-public/django-project-template/tree/spa)
+
```

Comment:
This package can be used directly as it also provides logic to detect if user has permissions or not.

Reply:
Added some examples for decorators.